src/source/source_cache.js

```diff
@@ -628,6 +647,43 @@ class SourceCache extends Evented {
         return retain;
     }
 
+    _updateLoadedParentTileCache() {
+        this._loadedParentTiles = {};
+
+        for (const tile of (Object.values(this._tiles): any)) {
```

Comment:
Unfortunately `Object.values` is not supported in IE11. A `for in` loop will do though.

Reply:
This is a good catch! Do we have any automatic ways of validating pull requests against different browsers?

Reply:
@mpulkki-mapbox I explored options for linting things like this, but it's quite difficult — e.g. while you can catch things like `Object.values`, many other things IE11 doesn't support like `array.find` are impossible to catch because they're instance methods and linting plugins can't derive object types.